components/skus/common/features.cc

```diff
@@ -0,0 +1,17 @@
+/* Copyright (c) 2021 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at http://mozilla.org/MPL/2.0/. */
+
+#include "brave/components/skus/common/features.h"
+
+#include "base/feature_list.h"
+
+namespace skus {
+namespace features {
+
+const base::Feature kSkusFeature{"SkusFeature",
+                                 base::FEATURE_DISABLED_BY_DEFAULT};
```

Comment:
This will have to be enabled by default for iOS

Reply:
Fixed in https://github.com/brave/brave-core/pull/10847/commits/2d6ef57d9ff97f291e0975858df498e32973c7b4